configs/templates/MetricTemplate.proto

```diff
@@ -8,62 +8,7 @@ import "mixer/configs/mixer/TemplateExtensions.proto";
 option (istio.mixer.v1.config.template.template_variety) = TEMPLATE_VARIETY_REPORT;
 option (istio.mixer.v1.config.template.template_name) = "Metric";
 
-message Type {
+message Constructor {
```

Comment:
hmm..Like `Template`. Let's call this `Template`, as this is going to be a schema which is going to be throw away and from which other concrete things emerge.